test/integration/functional_test.go

```diff
@@ -639,6 +639,30 @@ func validateProfileCmd(ctx context.Context, t *testing.T, profile string) {
 
 // validateServiceCmd asserts basic "service" command functionality
 func validateServiceCmd(ctx context.Context, t *testing.T, profile string) {
+	defer func() {
+		if t.Failed() {
+			t.Logf("service test failed - dumping debug information")
+
+			rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "describe", "po", "hello-node"))
```

Comment:
you probably want to do describe svc
"kubectl describe svc hello-minikub"

Reply:
It does so a few lines below.